lib/packages-dependencies.php

```diff
@@ -0,0 +1,235 @@
+<?php
+/**
+ * Returns array with the list of WordPress packages and their dependencies.
+ *
+ * @package gutenberg
+ */
+
+return array(
```

Comment:
I'd not known you could return from a file in PHP to be used in a variable assignment. Pretty neat, though I could imagine the non-traditional nature of it could be viewed by some as a negative. Unsure there's any convention around it. I don't find it particularly offensive; it seems to serve a similar purpose to `require( './foo.json' )` in Node. As purely static data, it could make sense as JSON, but I don't know that PHP provides any options for easily evaluating a JSON file to its equivalent array form. At least not without jumping through potential non-performant hoops involving `file_get_contents` and `json_decode`.

http://www.php.net/manual/en/function.return.php

Reply:
This is a very common technique in various PHP frameworks, where you optimize JSON or YAML config files by caching them as an array. I have no idea whether there are some patterns for that in core but I used that in the past many times.